tools/speech_data_explorer/data_explorer.py

```diff
@@ -187,6 +308,52 @@ def plot_histogram(data, key, label):
         ],
         className='bg-light rounded-bottom border-bottom border-left border-right',
     ),
+]
+if metrics_available:
+    stats_layout += [
+        dbc.Row(
+            [
+                dbc.Col(html.Div('Overall WER, %', className='text-secondary'), width=3, className='border-right'),
+                dbc.Col(html.Div('Overall CER, %', className='text-secondary'), width=3, className='border-right'),
+                dbc.Col(html.Div('Overall WMR, %', className='text-secondary'), width=3, className='border-right'),
+                dbc.Col(html.Div('Mean Word Accuracy, %', className='text-secondary'), width=3),
```

Comment:
What does `WMR` stand for? Maybe write the full name + the short name for the metrics of the first tab?